src/main/kotlin/racingcar/controller/GameController.kt

```diff
@@ -0,0 +1,60 @@
+package racingcar.controller
+
+import racingcar.model.Car
+import racingcar.model.Cars
+import racingcar.model.Repetition
+import racingcar.model.Winner
+import racingcar.utils.Round
+import racingcar.view.InputView
+import racingcar.view.OutputView
+
+class GameController {
+    private val inputView: InputView = InputView()
+    private val outputView: OutputView = OutputView()
```

Comment:
Dependency Injection을 적용하기 위해 아래와 같이 바꿔보는 건 어떨까요?

```suggestion
class GameController(
    private val inputView: InputView,
    private val outputView: OutputView,
) {
```

https://medium.com/backticks-tildes/the-s-o-l-i-d-principles-in-pictures-b34ce2f1e898
https://medium.com/free-code-camp/a-quick-intro-to-dependency-injection-what-it-is-and-when-to-use-it-7578c84fa88f

DI를 사용하는 이유를 체감하기 위해서는 Inversion of control을 적용해봐야 합니다.
3주차에는 아래 개념을 학습한 후, 각자 미션을 진행할 때 적용해보고, 코드 리뷰를 하면서 서로 이해한 개념을 점검해보는 것 어떤가요?😃
1. SOLID 원칙을 지킨다.
2. Inversion of control을 적용한다.( 단, 미션 구조상 적용할 수 없거나, 배보다 배꼽이 커지는 경우라면 하지 않는다. )

Reply:
1. SOLID 원칙에 대해 들어보기만 했는데, 프리코스 이후에 라도 왜 사용하는지 학습해보면 좋을 것 같네요.
2. 현재는 IOC 장점에 대해서만 간략하게 봤는데 의존성을 낮춰 재사용 가능한 코드를 만들 수 있다는 점이 매력적인 것 같습니다.

우테코에서 지향하는 3주차 학습 목표와 기타 요구 사항에 대해 완벽히 끝낸 뒤에 리팩토링 하는 방법으로 수행할 계획입니다. 따라서 시간이 부족하다면 프리코스 이후에 학습하게 될 것 같습니다. 저는 저번 주차의 피드백 반영과 금주의 목표만 따라가도 시간이 빠듯해서 어려울 것 같습니다. ㅜㅜ,,,